src/lib/default-type-to-mock.ts

```diff
@@ -16,6 +16,14 @@
 import ts from 'typescript';
 import {fake} from './fake';
 
+export enum SupportedTypes {
```

Comment:
Is this exhaustive of all the supported types of Intermock? There's quite a few other TS features that Intermock supports. Can you add a comment for what this is since it's not clear yet what this is for?

Reply:
Done